### PR TITLE
Super duper secret.

### PR DESCRIPTION
**Pull request by dashed**

### CODE REVIEW

File: src/sentry/utils/snuba.py [hunk 1]
```diff
@@ -926,16 +928,33 @@ def resolve_snuba_aliases(snuba_filter, resolve_func, function_translations=None
...
+            if aggregation[1].startswith("metrics."):
+                found = False
+                for col in selected_columns:
+                    if col[2].strip("`") == aggregation[1]:
+                        found = True
+                        continue
+                if not found:
+                    aggregation[
+                        0
+                    ] += "(toFloat32OrNull(tags.value[indexOf(tags.key, '{}')]))".format(
```
Comment (JTCunning) [inline src/sentry/utils/snuba.py:946]: I know that this is experimental.
With that being said, anyone who steps has concerns about snuba being slow on requesting these columns will just get linked this comment.

Comment (dashed) [reply src/sentry/utils/snuba.py:946]: Sounds good. 

This entire PR is not expected to be merged into sentry.

Comment (JTCunning) [reply src/sentry/utils/snuba.py:946]: ssh, ssh, keep your voice down. it's a super duper secret. they'll hear you 👀

Comment (fpacifici) [reply src/sentry/utils/snuba.py:946]: wait, so I was not supposed to post this on reddit ?

Comment (billyvg) [reply src/sentry/utils/snuba.py:946]: damn is it hackweek already

**Pull request closed without merging**